File: src/main/java/no/nav/sbl/soknadsosialhjelp/json/JsonSosialhjelpObjectMapper.java
package no.nav.sbl.soknadsosialhjelp.json;

import tools.jackson.databind.json.JsonMapper;

public final class JsonSosialhjelpObjectMapper {

    private JsonSosialhjelpObjectMapper() {
    }

    public static JsonMapper createObjectMapper() {
        return createJsonMapperBuilder().build();
    }

    public static JsonMapper.Builder createJsonMapperBuilder() {
        return JsonMapper.builder();
    }
}

File: src/main/kotlin/no/nav/sbl/soknadsosialhjelp/json/JsonSosialhjelpObjectMapper.kt
package no.nav.sbl.soknadsosialhjelp.json

import tools.jackson.databind.json.JsonMapper

object JsonSosialhjelpObjectMapper {
    @JvmStatic fun createObjectMapper(): JsonMapper = createJsonMapperBuilder().build()
    @JvmStatic fun createJsonMapperBuilder(): JsonMapper.Builder = JsonMapper.builder()
}
